devops/jobs/StaffSuperAccountAudit.groovy

```diff
@@ -0,0 +1,88 @@
+/*
+ Variables without defaults are marked (required) 
+ 
+ Variables consumed for this job:
+    * DEPLOYMENTS: (required)
+          deployment:
+            environments:
+              -  environment
+    * NOTIFY_ON_FAILURE: alert@example.com
+    * FOLER_NAME: folder
+    * RECIPIENTS: name@example.com (required)
+
+ This job expects the following credentials to be defined on the folder
+    ${environment}-${deployment}-mysql-credentials: secret file that contains the mysql credentials including user, password, and host
+
+*/
+
+package devops.jobs
+
+import static org.edx.jenkins.dsl.Constants.common_wrappers
+
+class StaffSuperAccountAudit {
+    public static def job = { dslFactory, extraVars ->
+        assert extraVars.containsKey('DEPLOYMENTS') : "Please define DEPLOYMENTS. It should be a list of strings."
+        assert !(extraVars.get('DEPLOYMENTS') instanceof String) : "Make sure DEPLOYMENTS is a list and not a string"
+        extraVars.get('DEPLOYMENTS').each { deployment, configuration ->
+            configuration.environments.each { environment ->
```

Comment:
How close are we to being able to write some sort of intermediate module that knows how to loop over out DEPLOYMENTS/ENVIRONMENTS lists?  I know you've done this in a few places now. 

This is not a chance to make in this PR, I just want you to think about the refactor and whether it's something we can do on the next job.